src/hooks/useCds/translate.js

```diff
@@ -209,12 +304,72 @@ function mapResult(result, loincMapping, testCodeResultMapping) {
 }
 
 //
-// EpisodeOfCare
+// Strides Data
 //
-function translateEpisodeOfCare(patientData) {
-  patientData
-    .filter(resource => resource.resourceType === 'EpisodeOfCare')
-    .forEach(episodeOfCare => mapEpisodeOfCare(episodeOfCare));
+function mapStrideResult(patientData, patientDataMap, stridesData) {
+  const mrn = patientDataMap.Patient[0].identifier?.find(id => id.type?.text === 'MRN')?.value;
```

Comment:
```suggestion
  const mrn = patientData.Patient ? patientDataMap.Patient[0]?.identifier?.find(id => id.type?.text === 'MRN')?.value : null;
```
If I run the dashboard with the STRIDES data loaded, I get the following error:

![Strides Mapping Error](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/assets/51176978/ba61b360-7321-49bc-b35b-b9c9f5e5405c)

Upon the dashboard's first load, the patientData array is empty, so line 310 is throwing an error. Something like what I suggested above should work, although you may be able to add handling higher in the file so that `mapStrideResult` is not called at all until all the patient data is loaded.

Reply:
Nice! I see that this was fixed by adding empty array and null checks at the top of the `mapStrideResult` and `patientDataToHash`. I retested and this situation no longer throws this error in the browser.